lib/Account.php

```diff
@@ -37,6 +37,11 @@
  */
 class Account extends ApiResource
 {
+    use ApiOperations\Create;
+    use ApiOperations\Update;
+    use ApiOperations\Delete;
+    use ApiOperations\All;
```

Comment:
Minor, but maybbeeee consider alphabetizing this lists just so that we have stable sorting if new traits are added down the line.

Reply:
Yeah, good point. Done :)